src/spells3.cc

```diff
@@ -3561,8 +3561,10 @@ const char *device_haste_monster_info()
 
 casting_result device_wish()
```

Comment:
Hum, can't figure out where to put this comment, but I think "wish" should perhaps be mentioned in the commit subject?

Reply:
You can control that when merging